plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/MavenFacet.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.maven.internal;
+
+import org.sonatype.nexus.repository.Facet;
+import org.sonatype.nexus.repository.maven.internal.policy.RedeployPolicy;
+import org.sonatype.nexus.repository.maven.internal.policy.VersionPolicy;
+
+/**
+ * Maven facet, present on all Maven repositories.
+ *
+ * @since 3.0
+ */
+@Facet.Exposed
+public interface MavenFacet
+    extends Facet
+{
+  VersionPolicy getVersionPolicy();
+
+  RedeployPolicy getRedeployPolicy();
```

Comment:
As mentioned elsewhere, redeployment policy seems like its a general aspect to all repos not specific to maven.  Ie. any repository could allow redeployment or only allow a single deployment?